src/Stepper/Stepper.tsx

```diff
@@ -37,8 +33,8 @@ export default class Stepper extends React.PureComponent {
   static defaultProps = defaultProps;
   static cssClass = cssClass;
 
-  _renderSidebar(style) {
-    const {currentStep, steps, onStepClick, className} = this.props;
+  render() {
+    const {currentStep, style, steps, onStepClick, className} = this.props;
```

Comment:
supporting `style` was only necessary before, when we expected `Sticky` to manage the component's style. can remove now to encourage people to keep styling in LESS for now